internal/bzlmod/go_mod.bzl

```diff
@@ -16,6 +16,94 @@ visibility([
     "//tests/bzlmod/...",
 ])
 
+def _validate_go_version(path, state, tokens, line_no):
+    if len(tokens) == 1:
+        fail("{}:{}: expected another token after 'go'".format(path, line_no))
+    if state["go"] != None:
+        fail("{}:{}: unexpected second 'go' directive".format(path, line_no))
+    if len(tokens) > 2:
+        fail("{}:{}: unexpected token '{}' after '{}'".format(path, line_no, tokens[2], tokens[1]))
+
+def use_spec_to_label(workspace_name, use_directive):
```

Comment:
Let's call the parameter `repo_name`. `Label#workspace_name` is deprecated and the replacement is also called `repo_name`.